pkg/dsdk/remote_provider.go

```diff
@@ -229,3 +229,27 @@ func (e *RemoteProvider) Reload(ro *RemoteProviderReloadRequest) (*RemoteProvide
 	RegisterRemoteProviderEndpoints(resp)
 	return resp, nil, nil
 }
+
+type RemoteProviderOperationsUpdateRequest struct {
```

Comment:
Minor nit: could we rename this to `RemoteProviderOperationSetRequest` and the func to `SetOperation` to maintain consistency with the other PUT structs/methods?